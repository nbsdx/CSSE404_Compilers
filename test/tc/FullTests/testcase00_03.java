// class not defined

class A
{
   public static void main( String[] args )
   {
      System.out.println( 0 );
   }
}

class B
{
   public boolean start( C c )
   {
      return false;
   }
}

