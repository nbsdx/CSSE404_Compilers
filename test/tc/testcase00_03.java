// class not defined for parameter

class A
{
   public static void main( String[] args )
   {
      System.out.println( 0 );
   }
}

class B
{
   public boolean start( C c )
   {
      return false;
   }
}

