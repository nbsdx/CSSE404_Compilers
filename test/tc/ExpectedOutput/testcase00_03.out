Cannot find class named C
